pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools",
+    "cython>=0.24.0",
+    "numpy<v1.20.0",
```

Comment:
FYI: this line is necessary to prevent `ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject`.

Ref: https://github.com/ymd-h/cpprb/discussions/3